src/common/WorkQueue.h

```diff
@@ -434,7 +434,7 @@ class ThreadPool : public md_config_obs_t {
   };
 private:
   vector<WorkQueue_*> work_queues;
-  int last_work_queue;
+  int next_work_queue;
```

Comment:
that's why i'd suggest do this in this very commit to avoid extra merge commit, and code churn. personally, i don't really like to give the source a overhaul to obscure the history just to use the new syntax, but we can always update the existing code base if we happen to touch a part of it: it's safer and it introduces less code churn.